proto/identity/associations/signature.proto

```diff
@@ -28,6 +31,8 @@ message Erc1271Signature {
   uint64 block_number = 2;
   // The actual signature bytes
   bytes signature = 3;
+  // RPC URL of the node where the contract is deployed
+  string chain_rpc_url = 4;
```

Comment:
Usually with RPC providers this would be handled at the app level, since the RPC URL might include secrets that are scoped to the app (like an Infura API key).

These signatures are public as part of your Inbox Log, and I don't think apps are going to be happy sharing these kinds of RPC URLs amongst each other.